### ISSUE TITLE
Apply app theme to launcher widget

### ISSUE THREAD

**Issue by 532910** [gsantner/markor#727]
![IMG_20191008_044002](https://user-images.githubusercontent.com/14617699/66361020-da114d80-e985-11e9-9840-234bf4bf25aa.jpg)

**Comment by stefanfris** [comment]
Hello if i understand this correctly, white background should be in other darker color? Im considering to take this as my first task ever, if it will be possible.  thanks

**Comment by 532910** [comment]
> white background should be in other darker color

together with the gray text color, that should be in other brighter color

**Comment by 532910** [comment]
And not only text color and background should be changes, but also horizontal lines and gray file icons.

**Comment by stefanfris** [comment]
Oh, okay now I understand. It isn't labelled with beginner or good first issue, so do you think is it suitable task for beginner?  think I'll be able to do that.

**Comment by 532910** [comment]
Looks like it should not be difficult.

**Comment by stefanfris** [comment]
Okay great,  so can you please assign it to me?

**Comment by 532910** [comment]
I have no such permissions, you should ask @gsantner for this.

**Comment by stefanfris** [comment]
Hi @gsantner would like to try solve this issue #727. Could you please assign it to me? Thanks.

**Comment by gsantner** [comment]
@stefanfris 
Thanks, sure go for it!
The files required to work on are located in the thirdparty writeley folder.

Check AppSettings::isDarkThemeEnabled to decide on the color/theme.

![grafik](https://user-images.githubusercontent.com/6735650/66395121-2ed4b880-e9c6-11e9-8228-bea93e243f1b.png)

![grafik](https://user-images.githubusercontent.com/6735650/66395129-34ca9980-e9c6-11e9-9141-e8f664eae261.png)

**Comment by 532910** [comment]
Not fixed, please reopen.

Moreover, it's unusable now:
![image](https://user-images.githubusercontent.com/14617699/71534392-9f2d0f00-290f-11ea-9c86-fd15ec792ec0.png)

**Comment by 532910** [comment]
Any update? It's not fixed, please reopen!

**Comment by gsantner** [comment]
Notice for all widget issues and feature requests: I'm not actively working on the widget (and never really have). I'm fully waiting for community contribution on this.

Beside PR review and tests by me, full community work on the widget feature is expected. So if you read this, please go for it, thanks. :smiley:.

**Comment by gsantner** [comment]
Thanks to @stefanfris who worked on widgets recently. Do you want to take a look into widgets again?
Would be very nice, thanks!

**Comment by 532910** [comment]
> Notice for all widget issues and feature requests: I'm not actively working on the widget (and never really have). I'm fully waiting for community contribution on this.

It's ok!

But the issue should be opened untill fixed. Do not treat it like a your responsibility. Treat it like a public todo list. It's not done, so should be unchecked (opened).

**Comment by el416** [comment]
Hello, is this issue still open? If so, could you assign it to me?

**Comment by el416** [comment]
Hi @gsantner , I have a working fix for this problem, could you assign this issue to me so I can create a PR afterwards?

**Comment by gsantner** [comment]
Thanks! You can make the PR without assignment, makes no difference

**Comment by el416** [comment]
Hi @gsantner , I made the pull request. 

I also tried to change the horizontal lines between individual notes to a different color, but the only way I managed to do it was through a new widget_layout.xml file. That, however, would create a new issue, because after applying a new layout to the widget, you would need to fill the widget with the notes once again, which I was not able to do. So, after changing the app theme you would have to delete the old widget and create a new one (which would have text as well as horizontal lines in correct color). Because of that, I decided not to apply this change in the pull request I made. However, if you were interested, I could make a new pull request with the mentioned changes included (in case you wanted to open a new issue for that, so someone could fix it, if it's worth the trouble in your opinion).
